src/modules/blockchain/implementation/web3-service.js

```diff
@@ -597,7 +617,7 @@ class Web3Service {
     ) {
         let result;
         const gasPrice =
-            predefinedGasPrice ?? (await this.getGasPrice()) ?? this.convertToWei(20, 'gwei');
+            predefinedGasPrice ?? (await this.getGasPrice()) ?? this.convertToWei(20, 'wei');
```

Comment:
This should be 20 wei for NeuroWeb, but for Gnosis i think that is far from enough, 20 gwei should be safe for Gnosis. So I would completely handle gas price in neuroweb/gnosis implementation of getGasPrice